tests/ci/release.py

```diff
@@ -274,9 +282,15 @@ def _commit_cmake_contributors(self, version: ClickHouseVersion) -> None:
                 self.run(f"git diff '{self.CMAKE_PATH}' '{self.CONTRIBUTORS_PATH}'"),
             )
             self.run(f"git checkout '{self.CMAKE_PATH}' '{self.CONTRIBUTORS_PATH}'")
+
+    def _commit_cmake_contributors(
+        self, version: ClickHouseVersion, reset_tweak: bool = True
```

Comment:
I don't see any calls with `false` is this intended?

Reply:
Theoretically, if we would bring back tags `git tag -l v*-testing`, then the tweak needs to be preserved.

Not sure if it's necessary for now.